features/wasm-multi-memory.yml

```diff
@@ -0,0 +1,5 @@
+name: WebAssembly multi-memory
+description: An extension to WebAssembly to allow the use of multiple memories within a single Wasm module.
```

Comment:
`Wasm` looks kinda funny, could do `WASM` as you noted in the main feature description, but also could probably just remove it here.